src/pages/Mypage/components/Likes.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import ItemTitle from './ItemTitle';
+import LikesProduct from './LikesProduct';
+import './Likes.scss';
+
+const Likes = () => {
+  const [likesList, setLikesList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/subscribeData.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/subscribeData.json')
```

option 부분에 GET 요청일 때 별다른 body나 header 값이 없다면
생략이 가능합니다!